pkg/controller/quota/quota.go

```diff
@@ -102,6 +103,17 @@ func EnsureQuotaRequest(req router.Request, resp router.Response) error {
 	return nil
 }
 
+// addContainers adds the number of containers and accounts for the scale of each container.
+func addContainers(containers map[string]apiv1.Container, quotaRequest *adminv1.QuotaRequestInstance) {
+	for _, container := range containers {
+		scale := 1
+		if container.Scale != nil {
+			scale = int(*container.Scale)
+		}
+		quotaRequest.Spec.Resources.Containers += scale
```

Comment:
So if scale is unset, we assume scale = 1? Is that the default?

Reply:
Yeah, it operates under the same rules as the [`Replicas` section of a DeploymentSpec](https://pkg.go.dev/k8s.io/api@v0.27.3/apps/v1#DeploymentSpec.Replicas:~:text=//%20Number%20of%20desired%20pods.%20This%20is%20a%20pointer%20to%20distinguish%20between%20explicit%0A%09//%20zero%20and%20not%20specified.%20Defaults%20to%201.%0A%09//%20%2Boptional%0A%09Replicas%20*int32%20%60json%3A%22replicas%2Comitempty%22%20protobuf%3A%22varint%2C1%2Copt%2Cname%3Dreplicas%22%60). We use it [here](https://github.com/tylerslaton/runtime/blob/07f8216fc68a5ab3c9e979e0f217a0c9c5225c6c/pkg/controller/appdefinition/deploy.go#L675-L676), to be more specific.